src/components/ForecastDay.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import moment from 'moment';
+import { renderWeatherImg, toCelcius } from '../utils/weather.js';
+
+const ForecastDay = props => {
+  const { temp, weather, dt } = props.weather;
```

Comment:
flatten your props before they get here, and destructure all of them.

Reply:
done